### PR TITLE
add Class Animal and Cow

### PR DESCRIPTION
**Pull request by ingbeck**
Added a new class Animal und Cow (extends Animal). In class Cow, I implemented a method called 'milk()', which checks, if a cow is ready to milk (isMilkable) and milks it. After the cow is milked, it sets the boolean isMilkable to false.

### CODE REVIEW

File: src/main/java/com/github/ingbeck/Main.java [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Animal animal1 = new Animal("Rex", 5, "Bulldog");
+        Animal animal2 = new Animal("Pumba", 4, "Wild Boar");
```
Comment (jessinowski) [inline src/main/java/com/github/ingbeck/Main.java:6]: cute!

File: src/main/java/com/github/ingbeck/Animal.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package com.github.ingbeck;
+import java.util.Objects;
+
+public class Animal {
+    String name;
+    int age;
+    String breed;
```
Comment (R-Schreyer) [inline src/main/java/com/github/ingbeck/Animal.java:7]: make private

Comment (ingbeck) [reply src/main/java/com/github/ingbeck/Animal.java:7]: cool, thx

File: src/main/java/com/github/ingbeck/Animal.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package com.github.ingbeck;
+import java.util.Objects;
+
+public class Animal {
+    String name;
```
Comment (jessinowski) [inline src/main/java/com/github/ingbeck/Animal.java]: oh oh! make private!

Comment (jessinowski) [inline src/main/java/com/github/ingbeck/Animal.java]:
```suggestion
    private String name;
```

**Pull request merged**